NativeCharList/Assets/NativeString.cs

```diff
@@ -247,10 +437,14 @@ public static void OverWrite(ref NativeString A, int position, NativeString stam
             }
         }
 
-        public static explicit operator NativeString(string A)
-        {
-            return new NativeString(A.Length, Allocator.Persistent);
-        }
+        /// <summary>
+        /// Removed temporarily because this is too likely to introduce errors.
+        /// </summary>
+        /// <param name="A"></param>
+        //public static explicit operator NativeString(string A)
+        //{
+        //    return new NativeString(A.Length, Allocator.Persistent);
+        //}
 
```

Comment:
Remove permanently. Make an overloaded constructor instead.

Reply:
This is the right solution.  I'll make sure to do that later.